src/util/domain-icon.js

```diff
@@ -38,6 +38,9 @@ export default function domainIcon(domain, state) {
     case 'lock':
       return state && state === 'unlocked' ? 'mdi:lock-open' : 'mdi:lock';
 
+    case 'garage_door':
+      return state && state === 'open' ? 'mdi:arrow-up' : 'mdi:arrow-down';
```

Comment:
Instead of using the `state-card-toggle` UI, why not use generalize the rollershutter state card for both garage door and rollershutter?